src/ixdat/readers/zilien.py

```diff
@@ -162,150 +177,279 @@ def read(self, path_to_file, cls=ECMSMeasurement, name=None, **kwargs):
+
+        return column_series
+
+    def _create_series_objects(self, column_headers, names_and_units, data):
+        """Create an Ixdat series objects from a given portion of a dataset."""
+
+        series_objects = []
+        time_series = None
+
+        for column_number, column_header in enumerate(column_headers):
+            # Skip meta columns in the EC-lab dataset
+            if column_header in ("experiment_number", "technique_number"):
+                continue
+
+            column_data = data[:, column_number]
+
+            # Skip holes in the EC-lab dataset
+            if all(np.isnan(column_data)):
+                continue
+
+            # Form series kwargs
+            series_name, unit, standard_name = names_and_units[column_number]
             series_kwargs = {
                 "name": series_name,
                 "unit_name": unit,
                 "data": column_data,
             }
-            if column_header == "Time [s]":  # Form TimeSeries
-                column_series = TimeSeries(**series_kwargs, tstamp=timestamp)
-                last_time_series = column_series
+
+            # Create the series
+            if column_header in ("Time [s]", "time/s"):
+                series_object = TimeSeries(**series_kwargs, tstamp=self._timestamp)
+                time_series = series_object
             else:
-                column_series = ValueSeries(**series_kwargs, tseries=last_time_series)
+                assert (
+                    time_series is not None
+                ), "Time column must be first in a dataset series."
+                series_object = ValueSeries(**series_kwargs, tseries=time_series)
 
-            series.append(column_series)
+            series_objects.append(series_object)
 
-        return series, aliases
+        return series_objects
+
+    # --- UTILS ---
+
+    @staticmethod
+    def _get_series_splits(series_headers):
+        """Series names and their index pairs in the whole read dataset.
+
+        Args:
+            series_headers (list): Series name headers (even empty).
+
+        Returns:
+            list, list: List of tuples with index pairs
+            and a list with non-empty series name headers.
+        """
+        series_cut_indices = []
+        nonempty_headers = []
+
+        for index, series_header in enumerate(series_headers):
+            if series_header != "":
+                series_cut_indices.append(index)
+                nonempty_headers.append(series_header)
+
+        # create cut pairs
+        # zip_longest to have the last pair with the index of the last
+        # series header to the end
+        series_cut_indices = [
+            (begin, end)
+            for begin, end in zip_longest(series_cut_indices, series_cut_indices[1:])
+        ]
+
+        return series_cut_indices, nonempty_headers
+
+    @staticmethod
+    def _get_biologic_splits(technique_numbers):
+        """Index pairs of row splits in the biologic dataset columns.
```

Comment:
Please update this on the other docstrings where it ids not the casem to follow pep257 https://peps.python.org/pep-0257/ so describe what it does, not what it returns

Reply:
> Notes:
> 
>     Triple quotes are used even though the string fits on one line. This makes it easy to later expand it.
>     The closing quotes are on the same line as the opening quotes. This looks better for one-liners.
>     There’s no blank line either before or after the docstring.
>     The docstring is a phrase ending in a period. It prescribes the function or method’s effect as a command (“Do this”, “Return that”), not as a description; e.g. don’t write “Returns the pathname …”.
>     The one-line docstring should NOT be a “signature” reiterating the function/method parameters (which can be obtained by introspection). Don’t do: